apps/fluent-tester/src/FluentTester/TestComponents/Notification/NotificationTest.tsx

```diff
@@ -0,0 +1,139 @@
+import * as React from 'react';
+import { View, Text } from 'react-native';
+import { Test, TestSection, PlatformStatus } from '../Test';
+
+// const NotificationVariant string: 'primary' | 'neutral' | 'danger' | 'warning';
+enum NotificationVariant { primary = 1, neutral, danger, warning };
+
+const Notification = (props) => {
+  const variant = props.variant;
+  let startText = ''; 
+  let endText = '';
+
+  const styles: {[key: string]: any} = {
+    container: {
+      width: null,
+      borderRadius: 12,
+
+      flexDirection: 'row',
+      padding: 16,
+    },
+    start: {
+      fontSize: 16,
+    },
+    end: {
+      fontSize: 16,
+      fontWeight: '500',
+    },
+  };
+  
+  if (variant === 'primary') {
+    styles.container['backgroundColor'] = 'skyblue';
+    styles.container['justifyContent'] = 'space-between';
+    
+    startText = 'Mail Archived';
+    endText = 'Undo';
+  }
+  else if (variant === 'neutral') {
+    styles.container['backgroundColor'] = 'lightgrey';
+    styles.container['alignItems'] = 'center';
```

Comment:
I'd rather you only construct the style once, rather than create it once and then modify it. You could do something like:
```ts
let containerBackgroundColor = 'clear';
switch(variant) {
  case 'neutral:
    containerBackgroundColor = 'skyBlue';
    break;
  case 'warning':
    ....
}

let styles = StyleSheet.create({
  ...
  container: {
  ...
  backgroundColor: containerBackgroundColor;
  },
}

```